plugins/pubIds/doi/locale/es_ES/locale.xml

```diff
@@ -0,0 +1,65 @@
+	<message key="plugins.pubIds.doi.manager.settings.doiPrefix">Prefijo DOI</message>
+	<message key="plugins.pubIds.doi.manager.settings.doiPrefix.description"><![CDATA[El Prefijo DOI es asignado por agencias registradoras (p. ej., <a href="http://www.crossref.org" target="_new">Crossref</a>) y está en el formato 10.xxxx (p. ej., 10.1234):]]></message>
+	<message key="plugins.pubIds.doi.manager.settings.doiPrefixPattern">El prefijo DOI es obligatorio y debe estar en el formato 10.xxxx.</message>
+	<message key="plugins.pubIds.doi.manager.settings.doiSuffix">Sufijo DOI</message>
+	<message key="plugins.pubIds.doi.manager.settings.doiSuffix.description">El sufijo DOI puede tomar cualquier forma, pero debe ser único entre todos los objetos de publicación con el mismo prefijo DOI asignado:</message>
+	<message key="plugins.pubIds.doi.manager.settings.doiSuffixPattern">Usa el patrón introducido abajo para generar sufijos DOI. Usa %j para iniciales de revista, %v para el número del volumen, %i para el número de la publicación, %Y para el año, %a para el ID del artículo de OJS, %g para la galerada de OJS, %f para el ID del archivo de OJS, %p para el número de página y %x para "Indentificador Personalizado".</message>
+	<message key="plugins.pubIds.doi.manager.settings.doiSuffixPattern.example">Por ejemplo, vol%viss%ipp%p podría crear un DOI tal como 10.1234/vol3iss2pp230</message>
+	<message key="plugins.pubIds.doi.manager.settings.doiSuffixPattern.issues">para publicaciones</message>
+	<message key="plugins.pubIds.doi.manager.settings.doiSuffixPattern.submissions">para artículos</message>
+	<message key="plugins.pubIds.doi.manager.settings.doiSuffixPattern.representations">para galeradas</message>
+	<message key="plugins.pubIds.doi.manager.settings.doiSuffixDefault">Usar patrones por defecto.</message>
+	<message key="plugins.pubIds.doi.manager.settings.doiSuffixDefault.description"><![CDATA[%j.v%vi%i para publicaciones<br />%j.v%vi%i.%a para artículos<br />%j.v%vi%i.%a.g%g para galeradas.]]></message>
+	<message key="plugins.pubIds.doi.manager.settings.doiSuffixCustomIdentifier">Introduzca un sufijo DOI individual para cada elemento publicado. Encontrará un campo DOI adicional en la página de metadatos de cada elemento.</message>
+	<message key="plugins.pubIds.doi.manager.settings.doiIssueSuffixPatternRequired">Por favor, introduzca el patrón del sufijo DOI para publicaciones.</message>
+	<message key="plugins.pubIds.doi.manager.settings.doiSubmissionSuffixPatternRequired">Por favor, introduzca el patrón del sufijo DOI para artículos.</message>
+	<message key="plugins.pubIds.doi.manager.settings.doiRepresentationSuffixPatternRequired">Por favor, introduzca el patrón del sufijo DOI para galeradas.</message>
+	<message key="plugins.pubIds.doi.manager.settings.doiReassign">Reasignar DOIs</message>
+	<message key="plugins.pubIds.doi.manager.settings.doiReassign.confirm">¿Estás seguro de que quieres borrar todos los DOIs existentes?</message>
```

Comment:
To follow the translation guide and keep consistency:

<message key="plugins.pubIds.doi.manager.settings.doiReassign.confirm">¿Está seguro que quiere borrar todos los DOIs existentes?</message>